### ISSUE TITLE
Relax constrain on transformers

### ISSUE THREAD

**Issue by adinapoli** [ixmatus/snaplet-hslogger#1]
Hi there!

I wanted to give your library a try, but my stack imposes me to work against transformers 0.3.

Any reason your snaplet depends from transformers-0.4? Would be possible to relax such a constrain? Thanks!

Alfredo

**Comment by ixmatus** [comment]
Yeah, I haven't touched in a month or two but no reason not to relax the
constraint.
